content/en/docs/concepts/storage/volumes.md

```diff
@@ -975,6 +975,38 @@ Simultaneous writers are not allowed.
 See the [RBD example](https://github.com/kubernetes/examples/tree/master/volumes/rbd)
 for more details.
 
+#### RBD CSI migration {#rbd-csi-migration}
+
+{{< feature-state for_k8s_version="v1.23" state="alpha" >}}
+
+The `CSIMigration` feature for `RBD`, when enabled, redirects all plugin
+operations from the existing in-tree plugin to the `rbd.csi.ceph.com` {{<
+glossary_tooltip text="CSI" term_id="csi" >}} driver. In order to use this
+feature, the
+[CEPH CSI driver](https://github.com/ceph/ceph-csi)
```

Comment:
Nit, I think this wants to be "Ceph CSI" rather than "CEPH CSI"

Reply:
Done